app/controllers/booking_controller.py

```diff
@@ -0,0 +1,60 @@
+import os
+from . import controllers
+from cache import get_from_cache, set_to_cache
+from config import Config
+from app.utils.dbutil import DBUtil
+from flask import Flask, request, jsonify, render_template
+
+app = Flask('app')
+Config.DATABASE = os.path.join(app.root_path + '/db/', 'database.db')
+app.config.from_object(Config)
+db = DBUtil(app, False).get_instance(False)
+
+@controllers.route('/booking', methods=['GET', 'PUT', 'DELETE'])
+def book():
```

Comment:
There should be heavier comments to describe the difference between the get, put, and delete implementations, and which use case they are linked to so that when we integrate w/ the frontend we don't go in blind. Put TODOs to indicate that the patient_id will come not from the request, but the cookie (i believe this is what will happen)